stylesheets/style.scss

```diff
@@ -45,6 +45,20 @@ body {
 
   .table-content table {
     width: 100%;
+
+    a {
+      margin-right: 10px;
+    }
+
+    form.destroy-action {
+      display: inline-block;
+      margin-left: 30px;
+      a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 5